packages/camera-web/src/Camera/hooks/useUserMedia.ts

```diff
@@ -197,16 +206,29 @@ export function useUserMedia(
   const { handleError } = useMonitoring();
   const isActive = useRef(true);
 
+  let cameraPermissionState: PermissionStatus['state'] | null = null;
   useEffect(() => {
     return () => {
       isActive.current = false;
     };
   }, []);
 
-  const handleGetUserMediaError = (err: unknown) => {
+  const handleGetUserMediaError = async (err: unknown) => {
```

Comment:
Why did you put `async` here?

Reply:
previously added to use permission query inside that, The promise is removed but not the async. now removed